taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from taxi.models import Car, Manufacturer, Driver
 
-# Create your views here.
+
+def index(request):
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+
+    context = {"num_drivers": num_drivers,
+               "num_manufacturers": num_manufacturers,
+               "num_cars": num_cars,
+               }
```

Comment:
```suggestion
    context = {
            "num_drivers": num_drivers,
            "num_manufacturers": num_manufacturers,
            "num_cars": num_cars,
      }
```

Reformat these lines, please.

Reply:
+